lighthouse-plugin-publisher-ads/computed/ad-lantern-metric.js

```diff
@@ -83,7 +83,7 @@ function addEdges(graph) {
     }
     if (isGptTag(node.record.url) && node.record.resourceType === 'Script') {
```

Comment:
Do we need to worry about an isAmpTag() case?

Reply:
Not sure, I'll try changing this to `isGptOrAmpTag`

Reply:
I don't think this is required.